src/main/archetype/ui.tests.cypress/assembly-ui-test-docker-context.xml

```diff
@@ -0,0 +1,44 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+<!--
+ |  Copyright 2023 Adobe Systems Incorporated
+ |
+ |  Licensed under the Apache License, Version 2.0 (the "License");
+ |  you may not use this file except in compliance with the License.
+ |  You may obtain a copy of the License at
+ |
+ |      http://www.apache.org/licenses/LICENSE-2.0
+ |
+ |  Unless required by applicable law or agreed to in writing, software
+ |  distributed under the License is distributed on an "AS IS" BASIS,
+ |  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ |  See the License for the specific language governing permissions and
+ |  limitations under the License.
+-->
+
+<!--
+ | DO NOT MODIFY
+-->
+    <id>ui-test-docker-context</id>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <formats>
+        <format>tar.gz</format>
+    </formats>
+    <fileSets>
+        <fileSet>
+            <directory>${basedir}</directory>
+            <includes>
+                <include>Dockerfile</include>
+                <include>testing.properties</include> <!-- opt-in test module in Cloud Manager -->
```

Comment:
Can you check on this line + testing.properties?
I think in the story we defined that by default there is no test opt-in yet after generation of the tests

Reply:
From JIRA:

> Cypress Tests are still opt-in, not enabled by default in the CM pipeline.